error-messages/category.messages.js

```diff
@@ -2,10 +2,14 @@ const CATEGORY_NOT_FOUND = 'CATEGORY_NOT_FOUND';
 const CATEGORY_ALREADY_EXIST = 'CATEGORY_ALREADY_EXIST';
 const CATEGORY_IS_NOT_MAIN = 'CATEGORY_IS_NOT_MAIN';
 const WRONG_CATEGORY_DATA = 'WRONG_CATEGORY_DATA';
+const CATEGORY_INPUT_ERROR = 'CATEGORY_INPUT_ERROR';
```

Comment:
Please, make this message more meaningful because currently, it says nothing about the root cause of error